### ISSUE TITLE
Old v1 requirements files do not get installed

### ISSUE THREAD

**Issue by geerlingguy** [ansible-community/ansible-lint#1376]
##### Summary

For my mac-dev-playbook repository (see [issue](https://github.com/geerlingguy/mac-dev-playbook/issues/91)), I have a requirements.yml file in the format:

```yaml
---
- name: geerlingguy.dotfiles
- name: geerlingguy.homebrew
- name: geerlingguy.mas
```

With ansible-lint 5.0.1, I was seeing this behavior (silent failure):

```
 10:14:19 Development/GitHub/mac-dev-playbook 
$ ansible-lint 
Running ansible-galaxy install --roles-path .cache/roles -vr requirements.yml
Running ansible-galaxy collection install -p .cache/collections -vr requirements.yml

 10:14:47 Development/GitHub/mac-dev-playbook 
$ echo $?
1
```

I found that I had to change the format of my requirements.yml file to have an explicit `roles` key so it would work:

```
---
roles:
  - name: geerlingguy.dotfiles
  - name: geerlingguy.homebrew
  - name: geerlingguy.mas
```

In ansible though, if I just do an `ansible-galaxy role install -r requirements.yml`, either file will work. So maybe the galaxy install command needs to be modified to make it work with the old-style requirements file?

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
ansible --version
ansible 2.10.5
  config file = /Users/jgeerling/Development/GitHub/mac-dev-playbook/ansible.cfg
  configured module search path = ['/Users/jgeerling/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.9/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.9.1 (default, Dec 17 2020, 10:08:12) [Clang 12.0.0 (clang-1200.0.32.27)]

ansible-lint --version
ansible-lint 5.0.1
```

- ansible installation method: pip
- ansible-lint installation method: pip

##### OS / ENVIRONMENT

macOS, Ubuntu Linux

##### STEPS TO REPRODUCE

See: https://github.com/geerlingguy/mac-dev-playbook/issues/91

And see fix in that repo: https://github.com/geerlingguy/mac-dev-playbook/commit/9bd4b41a2eb7e54ab07249fc102e845b305f1eb7

**Comment by geerlingguy** [comment]
@ssbarnea - I wouldn't call it "old v1" — there are literally zillions of requirements files like that, and unless you use collections (99% of my playbooks don't, and in the wild for the five or six orgs I've worked with in the past year, none of theirs do yet either), there's no reason to change the format of the file to include the top-level keys...

**Comment by ssbarnea** [comment]
No need to worry, you will have the fix before monday.
